app/views/layouts/_font_loading.html.haml

```diff
@@ -1,12 +1,25 @@
 / We want to get the holding font rendered straight away
 %link{ rel: "preload", href: asset_path('poppins-v20-latin-regular.woff2'), as: "font", type: "font/woff2", crossorigin: :anonymous }
+%link{ rel: "preload", href: asset_path('poppins-v20-latin-600.woff2'), as: "font", type: "font/woff2", crossorigin: :anonymous }
```

Comment:
We still get a CLS on some pages due to the lack of bold font. This is now an extra step where we render a 600 weight font as 500/600/700. This will help on some pages (e.g. it stops CLS on the JS exercises). 

If we try and do the same beyond this (e.g. preload 500/600/700) we're going to start not having any value from this.

Note: `font-display: fallback` gives us 100ms to get the font loaded. The font is tiny so we should be in that window.